pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -1157,6 +1157,10 @@ func volumeAttachmentAdded(obj interface{}) {
 	log.Debugf("volumeAttachmentAdded: volume=%v", volAttach)
 	if volAttach.Status.Attached {
 
+		if volAttach.Spec.Source.PersistentVolumeName == nil {
+			log.Warnf("PersistentVolumeName is nil")
+			return
```

Comment:
This will skip processing VolumeAttachments for vcp-to-csi migrated volumes and not add it to `volumeNameToNodesMap` cache. Do we want such behavior?

Reply:
If the `volAttach.Spec.Source.PersistentVolumeName` is nil then we cannot populate the `volumeNameToNodesMap`. So, not sure if there is anything something else we do. Do you have any suggestion on how to proceed?